docs: Type operator (#103)
lenra-io/docs: src/markdown/features/data-management.md

| [{:target="_blank" rel="noopener"}$lte](https://www.mongodb.com/docs/manual/reference/operator/query/lte/)   | Matches values that are less than or equal to a specified value. |
| [{:target="_blank" rel="noopener"}$in](https://www.mongodb.com/docs/manual/reference/operator/query/in/)     | Matches any of the values specified in an array. | 
| [{:target="_blank" rel="noopener"}$nin](https://www.mongodb.com/docs/manual/reference/operator/query/nin/)   | Matches none of the values specified in an array. |

> | [{:target="_blank" rel="noopener"}$type](https://www.mongodb.com/docs/manual/reference/operator/query/type/)   | Matches all values that match the specified type. |

Reviewer: Just copy the [mongo doc](https://www.mongodb.com/docs/manual/reference/operator/query/) content since it's the same elements. The type operator is not in this category:
```suggestion
| [{:target="_blank" rel="noopener"}$type](https://www.mongodb.com/docs/manual/reference/operator/query/type/)   | Selects documents if a field is of the specified type. |
```
